components/core/src/clp/clo/CommandLineArguments.hpp

```diff
@@ -93,18 +113,24 @@ class CommandLineArguments : public CommandLineArgumentsBase {
 
     void print_basic_usage() const override;
 
-    // Variables
-    std::string m_mongodb_uri;
-    std::string m_mongodb_collection;
-    uint64_t m_batch_size;
+    // Search variables
     std::string m_archive_path;
     bool m_ignore_case;
     std::string m_search_string;
     std::string m_file_path;
     epochtime_t m_search_begin_ts, m_search_end_ts;
+
+    // Results cache output variables
+    std::string m_mongodb_uri;
+    std::string m_mongodb_collection;
+    uint64_t m_batch_size;
     uint64_t m_max_num_results;
 
-    // Search aggregation variables
+    // Network output variables
```

Comment:
Since you've organized everything else nicely, let's move this above the results cache output variables so they remain alphabetical in terms of the output handler.